includes/functions/functions_osh_update.php

```diff
@@ -122,7 +122,15 @@ function zen_update_orders_history($orders_id, $message = '', $updated_by = null
                     strip_tags($email_message) .
                     $status_text . $status_value_text .
                     OSH_EMAIL_TEXT_STATUS_PLEASE_REPLY;
-           
+
+                // Add in store specific order message
+                $email_order_message = defined('EMAIL_ORDER_UPDATE_MESSAGE') ? constant('EMAIL_ORDER_UPDATE_MESSAGE') : '';
+                $GLOBALS['zco_notifier']->notify('NOTIFY_ORDER_SET_ORDER_UPDATE_MESSAGE'); 
```

Comment:
Shouldn't the `$email_order_message` variable be passed to the notifier here? Otherwise modifying it is more complicated than it needs to be.

Reply:
The other notifier also has no parms - see includes/classes/order.php line 1123.

Reply:
@lat9 can you weigh in here?  I have no opinion.

Reply:
Ya, the other one was inside a class though.

Reply:
Granted, I guess since it isn't a class property there then it's gotta be modified by reaching in as a global .... and reaching across things as globals is always a bad practice. While we already do that a bit in Zen Cart I'm trying to avoid proliferating that problem as much as possible. Think: never use globals unless there's no other way.

Perhaps we need to modify that notifier call in both places. A separate PR is fine for that.

Reply:
But wouldn't that break the code of people using the notifier now?

Reply:
the notifiers do have 2 different names, so if this one had a parm and the original one didn't that would be ok.

Reply:
It's a brand-new notifier, so I'm guessing that its use at this point is _low_.   I agree with @drbyte that the `$email_order_message` should be passed as notification parameter 2 (so it can be modified) with the `$orders_id` as the first parameter to give an observer a bit of context.

I'll note, too, that all other notifications issued by `zen_update_orders_history` use events named `ZEN_UPDATE_ORDERS_HISTORY_*`.

> Perhaps we need to modify that notifier call in both places. A separate PR is fine for that.

_Both_ places?  Where's the other one?

Reply:
These two notifiers are sisters: NOTIFY_ORDER_SET_ORDER_UPDATE_MESSAGE, NOTIFY_ORDER_SET_ORDER_MESSAGE.

Reply:
Could we change the 'name' of the one issued by `zen_update_orders_history` so it's more indicative of where it's 'coming from'?

Reply:
Sure!  What is your preferred name?

Reply:
`ZEN_UPDATE_ORDERS_HISTORY_SET_ORDER_UPDATE_MESSAGE`